webrender/res/brush_image.glsl

```diff
@@ -60,6 +59,8 @@ void main(void) {
     vec4 color = vColor * texture(sColor1, vec3(uv, vUv.z)).r;
 #endif
 
-    oFragColor = color * do_clip();
+    return color;
 }
 #endif
+
+#include brush
```

Comment:
why are we including it at the end rather than at the start?

Reply:
Being lazy and avoiding a forward declaration of the brush_vs and brush_fs functions :) Fixed.